rocky/tests/conftest.py

```diff
@@ -820,3 +820,21 @@ def list_origins(
 @pytest.fixture
 def mock_octopoes_api_connector(valid_time):
     return MockOctopoesAPIConnector(valid_time)
+
+
+@pytest.fixture
+def listed_hostnames(network):
+    return [
+        Hostname(network=network.reference, name="example.com"),
+        Hostname(network=network.reference, name="a.example.com"),
+        Hostname(network=network.reference, name="b.example.com"),
+        Hostname(network=network.reference, name="c.example.com"),
+        Hostname(network=network.reference, name="d.example.com"),
+        Hostname(network=network.reference, name="e.example.com"),
+        Hostname(network=network.reference, name="f.example.com"),
+    ]
+
+
+@pytest.fixture
+def now_formatted(valid_time):
```

Comment:
This isn't really a good name. I also don't think it very useful use of a fixture, I think it is better to just put `valid_time.strftime("%Y-%m-%d")` in the test.